tests/components/wallbox/test_config_flow.py

```diff
@@ -24,29 +23,6 @@ async def test_show_set_form(hass: HomeAssistant) -> None:
     assert result["step_id"] == "user"
 
 
-async def test_form_invalid_auth(hass):
-    """Test we handle invalid auth."""
-    result = await hass.config_entries.flow.async_init(
-        DOMAIN, context={"source": config_entries.SOURCE_USER}
-    )
-
-    with patch(
-        "homeassistant.components.wallbox.config_flow.WallboxHub.async_authenticate",
```

Comment:
This was patching the wrong function, we should have patched `"homeassistant.components.wallbox.config_flow.WallboxHub._authenticate"` instead.
However, the test was redundant, a 403 http error will cause an `InvalidAuth` exception to be raised, that's tested by `test_form_cannot_authenticate` just below.